tuxemon/states/items/__init__.py

```diff
@@ -112,7 +112,7 @@ def on_menu_selection(self, menu_item: MenuItem[Item]) -> None:
 
         """
         item = menu_item.game_object
-        state = self.determine_state_called_from()
+        states = [a.name for a in self.client.active_states]
```

Comment:
I feel like naming could be confusing here.

`states` is not a list of states, but a list of state **names**.
Maybe it could be renamed `active_state_names`?

Reply:
you're right, it could be confusing